cmake/EthDependencies.cmake

```diff
@@ -89,7 +82,7 @@ See https://developer.apple.com/library/mac/qa/qa1118/_index.html for more info.
 	if(MSVC)
 		set(CMAKE_FIND_LIBRARY_SUFFIXES .lib .a ${CMAKE_FIND_LIBRARY_SUFFIXES})
 	else()
-		set(CMAKE_FIND_LIBRARY_SUFFIXES .a )
+		set(CMAKE_FIND_LIBRARY_SUFFIXES .a .dylib)
```

Comment:
Not sure how the precedence works here. Will it prioritise .a over .dylib where both are present?